packages/extensions/modelerfour/src/quality-precheck/prechecker.ts

```diff
@@ -166,7 +166,7 @@ export class QualityPreChecker {
               const details = diff
                 .map((each) => `${each.path.join(".")} => '${each.op === "delete" ? "<removed>" : each.val}'`)
                 .join(",");
-              this.session.error(
+              this.session.warning(
```

Comment:
Are we sure all languages are able to handle this scenario?

Reply:
I talked with @lmazuel about this and the idea is to keep it as a warning to tell the user that this is indeed a warning and the redefinition of this type might not work. If the question is are generators able to handle the cases that should be working right now, I do think there might be some change required to support those scenarios. @kristapratico, is doing some of those changes on the python side and can confirm.

Reply:
If someone is trying to change the property type for inherited type using this feature in C# generator the generated code would plainly not work.

What are the cases we are targeting to support?

Reply:
I might be mixing different language here but I think there is 2 use cases:
- Having the child field type be inheriting the parent field type
- Having the child field have more restriction(validation) 

```cs
public class Parent
{
    public FormField Field { get; set; };
}

public class Child : Parent
{
    public StringFormField Field { get; set; };
}

public class FormField
{

}

public class StringFormField : FormField
{

}
```

Reply:
Yeah, the current C# generator doesn't support this case and it might not be trivial to implement.

Not blocking this PR just FYI.